integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/describeTable/TestDescribeTable.scala

```diff
@@ -68,15 +68,6 @@ class TestDescribeTable extends QueryTest with BeforeAndAfterAll {
     assert(descPar.exists(_.toString().contains("Partition Parameters:")))
   }
 
-  test(testName = "Compressor Type update from carbon properties") {
-    sql("drop table if exists b")
-    sql(sqlText = "create table b(a int,b string) STORED AS carbondata")
-    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.COMPRESSOR, "gzip")
-    val result = sql(sqlText = "desc formatted b").collect()
-    assert(result.filter(row => row.getString(0).contains("Data File Compressor")).head.getString
```

Comment:
As now we are not changing the default compressor, we can revert this change ?

Reply:
No, this print is not correct. For example, when loading, user set to use ZSTD compressor, and then set to SNAPPY again after loading, then this command still prints SNAPPY which is misleading